pulser-core/pulser/channels/dmm.py

```diff
@@ -124,7 +125,9 @@ def validate_pulse(
                 (defaults to a detuning map with weight 1.0).
         """
         super().validate_pulse(pulse)
-        round_detuning = round(pulse.detuning.samples, 6).as_array(detach=True)
+        round_detuning = pm.round(pulse.detuning.samples, 6).as_array(
+            detach=True
+        )
```

Comment:
What was the issue with the previous version?

Reply:
It gave me a mypy error related to incompatibility of types.